pkgs/development/interpreters/python/cpython/2.7/default.nix

```
@@ -234,8 +242,7 @@ in with passthru; stdenv.mkDerivation ({
         ln -s $out/lib/${libPrefix}/pdb.py $out/bin/pdb${sourceVersion.major}.${sourceVersion.minor}
         ln -s $out/share/man/man1/{python2.7.1.gz,python.1.gz}
 
-        # Python on Nix is not manylinux1 compatible. https://github.com/NixOS/nixpkgs/issues/18484
-        echo "manylinux1_compatible=False" >> $out/lib/${libPrefix}/_manylinux.py
+        ln -s ${manyLinux} $out/lib/${libPrefix}/_manylinux.py
```

```suggestion
        ln -s ${./_manylinux.py} $out/lib/${libPrefix}/_manylinux.py
```
Since the file has no dynamic components, it could be copied into nixpkgs.

---

I think that duplication is an unfortunate result of the duplicated python build expressions. I have the feeling that this duplication is still okay. If there would be yet another we should think of a solution. Also newer manylinux specifications might not be valid for python2.

Regardless of that I think we should use `cp` instead of `ln` since otherwise we create a depenency on a single file output for everything that depends on python while it might as well be copied in there.